jcl/src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -96,6 +97,14 @@ public StackTraceElement(String classLoaderName, String module, String version,
 public String getClassLoaderName() {
 	return classLoaderName;
 }
+
+/**
+ * Returns the ClassLoader used to load the class for the method in the stack frame.
+ * @return the ClassLoader or null
+ */
+public ClassLoader getClassLoader() {
```

Comment:
Public/protected methods which aren't part of the java specification can't be added. You can get access via a method in java.lang.VMAccess. I'm concerned about keeping a strong reference to the ClassLoader, possibly it may keep ClassLoader's from being GCed. I suggest computing isBuiltInClassLoader natively instead and just keep the answer. All the built in loaders are part of the J9JavaVM.

Reply:
Or maybe pass in isBuiltInClassLoader  to Util.printStackTraceElement() instead of using VMAccess.

Reply:
I moved the `isBuiltInClassLoader` check to `runtime/jcl/common/jclexception.c` and now store the result as a `boolean` in `StackTraceElement`, instead of the ClassLoader itself.

Would `isBuiltInClassLoader` be better placed as a utility function in util_api or vm_api?

Reply:
Unless you can find another place that needs this function (I can't), I think it's fine as is.